docs/rst/installation/linux_source_installation.rst

```diff
@@ -4,18 +4,114 @@ Linux installation from sources
 ===============================
 
 This section explains how to build Vulcanexus in Ubuntu Focal.
-Please, start following the `building instructions for ROS 2 Galactic Geochelone <https://docs.ros.org/en/galactic/Installation/Ubuntu-Development-Setup.html>`_ following the steps in the `System setup <https://docs.ros.org/en/galactic/Installation/Ubuntu-Development-Setup.html#system-setup>`_ and the `Get ROS 2 code <https://docs.ros.org/en/galactic/Installation/Ubuntu-Development-Setup.html#get-ros-2-code>`_ sections.
+Since Vulcanexus is a ROS 2 all-in-one tool set, certain ROS 2 prerequisites need to be met before building.
+
+ROS 2 prerequisites
+-------------------
+
+First of all, setup a `UTF-8` locale as required by ROS 2.
```

Comment:
```suggestion
First of all, set up a `UTF-8` locale as required by ROS 2.
```

This typo is also in the other section. Please, fix it in both installation guides.